wagtail/admin/viewsets/model.py

```diff
@@ -146,9 +146,11 @@ def get_index_view_kwargs(self, **kwargs):
             "search_fields": self.search_fields,
             "search_backend_name": self.search_backend_name,
             "paginate_by": self.list_per_page,
-            "default_ordering": self.ordering,
             **kwargs,
         }
+        if self.ordering:
+            view_kwargs["ordering"] = self.ordering
```

Comment:
Now that the view attribute is back to `default_ordering`, this ought to be:
```suggestion
            view_kwargs["default_ordering"] = self.ordering
```
It looks like the tests are currently passing "by accident" because this is overwriting the `ordering` cached_property, causing it to bypass the URL variable handling and always order on the field given in the viewset. I suggest we add an additional test to catch this, where we fetch `self.client.get(reverse("fctoy-alt3:index"), {"ordering": "-strid"})` and verify that the results are returned in reverse order.

Reply:
@gasman The tests were indeed passing by accident but I can't see where the viewset ordering information is getting lost. So I finally wrote a second test that I thought would surely work - passing the `ordering` parameter in the url. That test if broken too. I am either missing something simple OR there is a bug in some of the ordering stuff that is not related to the viewset change I am trying to make. I think it might be the later since the query args test fails even when I remove the changes I made to `get_index_view_kwargs`.